pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -4005,6 +4005,50 @@ func Test_server_Update(t *testing.T) {
 		}
 		return nil
 	}
+
+	/*
+		Update test cases (only test float32 unless otherwise specified):
+		- Equivalence Class Testing ( 1000 vectors inserted before a update )
+			- case 1.1: success update vector
```

Comment:
1 vector or all vectors?

Reply:
It means 1 vector. I will make it explicit.

Reply:
I think it is single update function to update 1 vector, we do not need to specify the number of the vector to be update.
What do you think?

Reply:
hmm...
I think update is an unary operation, so I agree with you that it is not necessary to describe the number of vectors.
However, the meaning does not change whether to specify or not, I think there is no problem to do it.

Reply:
> I think it is single update function to update 1 vector, we do not need to specify the number of the vector to be update.
But, 1000 vectors inserted before update test, right :sweat_smile: ?

Reply:
@kevindiu :arrow_up:

Reply:
Yes, I mean it is a update function to update only 1 vector, it is redundant to specific the number of vector to be update IMO.
both ok for me.